solr/core/src/java/org/apache/solr/handler/RealTimeGetHandler.java

```diff
@@ -51,7 +53,7 @@ public String getDescription() {
 
   @Override
   public Collection<Api> getApis() {
-    return ApiBag.wrapRequestHandlers(this, "core.RealtimeGet");
+    return Lists.newArrayList(AnnotatedApi.getApis(new RealTimeGetAPI(this)));
```

Comment:
Please avoid Guava for this; I think you can simply use List.of even though it's immutable (probably a good thing).  I know this PR is merged; my comment is for your future consideration.